Libs/MRML/Core/vtkMRMLSliceDisplayNode.h

```diff
@@ -93,6 +93,41 @@ class VTK_MRML_EXPORT vtkMRMLSliceDisplayNode : public vtkMRMLModelDisplayNode
   static const char* GetIntersectingSlicesInteractiveHandlesVisibilityModeAsString(int id);
   static int GetIntersectingSlicesInteractiveHandlesVisibilityModeFromString(const char*);
 
+  // Intersection mode
+  enum IntersectionMode
+  {
+    SkipLineCrossings = 1, // Lines do not intersect and there is a gap around the intersection point
+    FullLines = 2, // Lines intersect
+    IntersectionMode_Last // insert new types above this line
+  };
+  vtkGetMacro(IntersectingSlicesIntersectionMode, int);
+  vtkSetMacro(IntersectingSlicesIntersectionMode, int);
+
+  /// Return a string representing the intersection mode, set it from a string
+  const char* GetIntersectingSlicesIntersectionModeAsString();
+  void SetIntersectingSlicesIntersectionModeFromString(const char* intersectionModeString);
+
+  static const char* GetIntersectingSlicesIntersectionModeAsString(int id);
+  static int GetIntersectingSlicesIntersectionModeFromString(const char*);
+
+  // Line thickness mode
+  enum LineThicknessMode
+  {
+    FineLines = 1, // Thin lines
```

Comment:
Is there a specific reason for starting the values at 1 instead of 0?

Reply:
I assigned the value 1 to match the mapping set by @jamesobutler. In [qSlicerViewersToolBar_p.h](https://github.com/Slicer/Slicer/pull/6186/commits/66b5c2f41b3edbd39f5e13030ab346a2d5ac8dd8#diff-83f42288a06682e28ba5ca32083c95c00e9a774355c716b34829a5ad67a46e77R257), the fine action is mapped to `vtkMRMLCrosshairNode::Fine` (value = 1) , the medium action is mapped to `vtkMRMLCrosshairNode::Medium` (value = 2), and the thick action is mapped to `vtkMRMLCrosshairNode::Thick` (value = 3). That's why I decided to start values at 1.